bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/toolbar-submenu-item/component.jsx

```diff
@@ -50,18 +50,22 @@ export default class ToolbarSubmenuItem extends Component {
   }
 
   render() {
+
+    const { className, customIcon, icon, label } = this.props
+
     return (
       <div className={styles.buttonWrapper}>
         <Button
           hideLabel
           role="button"
-          color="default"
-          size="md"
-          label={this.props.label}
-          icon={this.props.icon}
-          customIcon={this.props.customIcon}
-          onMouseUp={this.handleOnMouseUp}
-          className={this.props.className}
+          color={'default'}
+          size={'md'}
```

Comment:
:+1: for making the props into constants
:-1: for switching from `"blah"` back to `{'blah'}`